src/template.yml

```diff
@@ -2426,6 +2495,11 @@ Resources:
               - iam:UntagRole
             Resource:
               - !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/adf-update-cross-account-access-role"
+          - Effect: "Allow"
+            Action:
+              - iam:*
+            Resource:
+              - "*"
```

Comment:
I assume this was added to debug something that didn't work.
We cannot add this, that would be way too permissive.